code/modules/sanity/breakdowns.dm

```diff
@@ -1,3 +1,5 @@
+#define BREAKDOWN_ALTERT_COOLDOWN rand(30 SECONDS, 120 SECONDS)
```

Comment:
`BREAKDOWN_ALTERT_COOLDOWN` is this meant to be `BREAKDOWN_ALERT_COOLDOWN`?

Reply:
I do not understand you. Do you suggest changing the name of this definition?

Reply:
What does ALTERT mean?